src/test/scala/org/broadinstitute/dsde/rawls/jobexec/MethodConfigResolverSpec.scala

```diff
@@ -117,29 +117,27 @@ class MethodConfigResolverSpec extends WordSpecLike with Matchers with TestDrive
   }
 
 
-  "MethodConfigResolver" should {
-    "resolve method config inputs" in withConfigData {
```

Comment:
This is a consequence of changing from `WordSpecLike` to `FlatSpecLike`, which was helpful making BeforeAndAfterAll apply to all tests in the same way